src/js/components/container/generic/MhubSyncingComponent.jsx

```diff
@@ -0,0 +1,38 @@
+import { Component } from 'react'
+import Messenger from '../../../services/messenger'
+import axios from 'axios'
+
+class RestSyncingComponent extends Component {
+
+  constructor(topic, urlPromise) {
+    super()
+    this.topic = `${topic}`
+    this.state = { data: '' }
+    this.urlPromise = urlPromise.then(url => this.url = url)
+  }
+
+  componentWillMount() {
+    this.load()
+    Messenger.on(this.topic, (data, msg) => this.reload(data, msg))
+  }
+
+  load () {
+    return this.urlPromise
+      .then(() => axios.get(this.url))
+      .then(response => {
+        this.setState({ data: response.data })
+      })
+      .catch(error => {
+        console.warn(error)
+        this.setState({ error })
+      })
+  }
+
+  reload(data, msg){
+    console.log(data,msg)
+    this.setState({ data: data.data.value })
```

Comment:
This can't be right `data.data`

Reply:
But it this.
I can change the name. But the object that mhub is returning have a data property